src/Packetery/Module/Framework/WcCartTrait.php

```diff
@@ -27,10 +27,11 @@ public function cartGetCartContents(): array {
 
 	/**
 	 * Gets cart total. This is the total of items in the cart, but after discounts. Subtotal is before discounts.
+	 * Really returns string, at least sometimes.
 	 *
-	 * @return float
+	 * @return float|string
 	 */
-	public function cartGetCartContentsTotal(): float {
+	public function cartGetCartContentsTotal() {
 		return WC()->cart->get_cart_contents_total();
```

Comment:
what's about `return (float) WC()->cart->get_cart_contents_total();` ?

Reply:
In my opinion adapter should not perform any logic and plugin should get raw values from the framework.

Reply:
ok, up to you.

Reply:
I'm voting for adding `(float)` => adapter can validate data, improve readability, phpdoc comments, native types etc.